### PR TITLE
docstring updates for mkdocs

### PR DESCRIPTION
**Pull request by cordeezy2**
docstring replacement for mkdocs

### CODE REVIEW

File: ipfabric/client.py [hunk 1]
```diff
@@ -117,31 +123,37 @@ def fetch_all(
...
 
     @check_format
-    def query(self, url: str, payload: Union[str, dict], all: bool = True):
-        """
-        Submits a query, does no formatting on the parameters.  Use for copy/pasting from the webpage.
-        :param url: str: Example: https://demo1.ipfabric.io/api/v1/tables/vlan/device-summary
-        :param payload: Union[str, dict]: Dictionary to submit in POST or can be JSON string (i.e. read from file).
-        :param all: bool: Default use pager to get all results and ignore pagination information in the payload
-        :return: list: List of Dictionary objects.
+    def query(self, url: str, payload: Union[str, dict], all: bool = True) -> list:
```
Comment (ghost) [inline ipfabric/client.py]: @cordeezy2 I guess I used `all` which is a builtin function.  What should we change this variable name to?

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -79,6 +79,27 @@ jobs:
           password: ${{ secrets.PYPI }}
     needs:
       - test
+
+  build-docs:
```
Comment (ghost) [inline .github/workflows/release.yml]: Shouldn't this not depend on releasing a new package?  What if there is an error in the docs we would have to make a whole new ipfabric version?

File: ipfabric/models.py [hunk 3]
```diff
@@ -12,7 +13,95 @@
 IGNORE_COLUMNS = {"id"}
 
 
+class Site(BaseModel):
```
Comment (ghost) [inline ipfabric/models.py]: This was all moved to `snapshot_models.py`  I don't think you merged correctly?

**Review by ghost — CHANGES_REQUESTED** [review]

File: ipfabric/client.py (hunk 1 above)
Comment (cordeezy2) [reply ipfabric/client.py]: `get_all` ?

Comment (cordeezy2) [reply ipfabric/client.py]:
```suggestion
    def query(self, url: str, payload: Union[str, dict], get_all: bool = True) -> list:
```

File: .github/workflows/release.yml (hunk 2 above)
Comment (cordeezy2) [reply .github/workflows/release.yml]: agreed, moved to its own yml file see commit https://github.com/community-fabric/python-ipfabric/pull/214/commits/fe3764421b0183bb50d2e38009bc4e6ef2fe1b69

File: ipfabric/models.py (hunk 3 above)
Comment (cordeezy2) [reply ipfabric/models.py]: fix in latest commit [7ff8abc](https://github.com/community-fabric/python-ipfabric/pull/214/commits/7ff8abca01f2572e0b002661a37ecc89f841731f)

**Comment by cordeezy2** [comment]
@justinjeffery-ipf created a new workflow, made it so docs build CI only runs when changes occur to docs repo. mkdocs.yml, or build_docs.yml

**Pull request merged**